src/randombytes.rs

```diff
@@ -1,9 +1,16 @@
 //! Cryptographic random number generation.
 
+#![warn(intra_doc_link_resolution_failure)]
```

Comment:
This is on by default in nightly, you don't need to add it explicitly.

Reply:
👍